modules/tinymce/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Flash of unstyled content while loading the editor because the content css was loaded after the editor content was rendered #TINY-7249
 - Unbinding an event handler did not take effect immediately while the event was firing #TINY-7436
 - Binding an event handler incorrectly took effect immediately while the event was firing #TINY-7436
+- Fixed RGBA values provided in the `color_map` setting given the wrong hex value when containing some transparency #TINY-7163
```

Comment:
```suggestion
- Partially transparent RGBA values provided in the `color_map` setting were given the wrong hex value #TINY-7163
```
Maybe something like this could flow smoother? I'm not sure if I actually preserved the meaning though